src/ImuSensor_TEST.cc

```diff
@@ -511,6 +516,174 @@ TEST(ImuSensor_TEST, CustomRpyParentFrame)
   EXPECT_EQ(defultOrientValue, sensor->Orientation());
 }
 
+sdf::ElementPtr generateSensor(std::string namedFrame)
```

Comment:
nit: I would rename this helper function to `sensorWithLocalization` and the `namedFrame` parameter to `_orientationLocalization`